source/agora/consensus/state/Ledger.d

```diff
@@ -587,6 +587,19 @@ public class Ledger
         }
     }
 
+    /// Returns: Whether the `utxo` can be used as a stake for a validator
+    public bool isStake (in Hash hash, in Output utxo) scope @safe
```

Comment:
`getPenaltyDeposit() > 0` would be equivalent

Reply:
I have questions about that name... It's what it seems to be doing but it is not at all clear what other role(s) it takes ?
Also this doesn't do the UTXO lookup.

Reply:
It has a history:) At first it was called `hasPenaltyDeposit` and returned a `bool`. But after I decided to let the melting TX spend the deposit, I needed a way to get how much the deposit was as well. Deposit is fixed and configured in `ConsensusParams` but our TX validation routine have no access to it.